jina/executors/crafters/frameworks.py

```diff
@@ -0,0 +1,17 @@
+from jina.executors.crafters import BaseSegmenter
+from jina.executors.devices import TorchDevice
+
+
+class BaseTorchSegmenter(TorchDevice, BaseSegmenter):
+
+    def __init__(self, model_name: str = None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.model_name = model_name
```

Comment:
I also recommend completely remove this class, as the only thing we inherit is `model_name`, which is not a torch/segment related pattern and there is no clue that this pattern is general.

```suggestion
class BaseTorchSegmenter(TorchDevice, BaseSegmenter):

    def __init__(self, model_name: str = None, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.model_name = model_name
```